docroot/sites/all/modules/custom/midcamp_eventbrite/includes/AttendeeProcessor.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+class AttendeeProcessor {
+  protected $attendees = array();
+
+  public function __construct(array $attendees = array()) {
+    if (empty($attendees)) {
+      $this->fetchAttendees();
+    }
+  }
+
+  public function process() {
+    foreach ($this->attendees as $attendee) {
+      $user = $this->loadUserByAttendee($attendee);
+      $conference_profile = $this->conferenceProfileSync($user, $attendee);
+      $order_profile = $this->orderProfileSync($user, $attendee);
+    }
+  }
+
+  protected function fetchAttendees() {
+    $response = EventbriteApi::request('GET', EVENTBRITE_API_ENDPOINT . '/events/' . EventbriteApi::eventId() . '/attendees');
```

Comment:
It use to be the order object came with this call. Now it's considered an "expansion". See https://www.eventbrite.com/developer/v3/formats/attendee/#ebapi-std:format-attendee

This line might need to be `$response = EventbriteApi::request('GET', EVENTBRITE_API_ENDPOINT . '/events/' . EventbriteApi::eventId() . '/attendees?expand=order');`